docs/mixin/dashboards/promscale.json

```diff
@@ -1774,7 +1774,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "exemplar": true,
-          "expr": "1 - promscale_sql_database_health_check_errors_total / promscale_sql_database_health_check_total",
+          "expr": "100.0 - (\n  sum(rate(promscale_sql_database_health_check_errors_total[$__rate_interval])) by (instance)\n  /\n  sum(rate(promscale_sql_database_health_check_total[$__rate_interval])) by (instance)\n) * 100",
```

Comment:
@arajkumar can you double check this. The linter was giving the following error:

```
[target-counter-agg-rule] 'Promscale': Dashboard 'Promscale', panel 'Database health', target idx '0' counter metric 'promscale_sql_database_health_check_errors_total' is not aggregated with rate, irate, or increase
```

Reply:
I fixed in https://github.com/timescale/promscale/pull/1688/files#diff-bee96e42fa9d10b4bc5817498d1cb0e50604815cb114e39afa9f56a9b0721217R1777. You can use it if you want.